src-docs/src/views/selectable/selectable_example.js

```diff
@@ -185,64 +187,31 @@ export const SelectableExample = {
           </p>
         </Fragment>
       ),
-      props: { EuiSelectable },
+      props,
       demo: <SelectableSingle />,
-      snippet: `
-      <EuiSelectable
+      snippet: `<EuiSelectable
```

Comment:
Use an array here for the multiple snippets
```suggestion
      snippet: [`<EuiSelectable
```

Reply:
https://github.com/elastic/eui/commit/fc001b21dddc0f74bb44fef4564c9d42ba353f15